src/app/dashboard/accounts/[guid]/page.tsx

```diff
@@ -94,11 +93,6 @@ export default function AccountPage({ params }: AccountPageProps): JSX.Element {
     return true;
   });
 
-  const split1 = new Split();
-  split1.fk_account = account;
-
-  const split2 = new Split();
```

Comment:
the split was created in the first render and then reused for every transaction you add from this page which is bad bad.